wallet/src/de/schildbach/wallet/ui/send/SendCoinsFragment.kt

```diff
@@ -106,6 +106,10 @@ class SendCoinsFragment: Fragment(R.layout.send_coins_fragment) {
                 .add(R.id.enter_amount_fragment_placeholder, fragment)
                 .commitNow()
             enterAmountFragment = fragment
+        } else {
+            enterAmountFragment = childFragmentManager.findFragmentById(
+                R.id.enter_amount_fragment_placeholder
+            ) as EnterAmountFragment
```

Comment:
If `savedInstanceState` isn't null we still want to call `initPaymentIntent` to restore the intent from the saved state, although it is still not necessary to do a `EnterAmountFragment` transaction - only to restore it into the field value.